curriculumBuilder/settings.py

```diff
@@ -126,8 +126,11 @@
 LANGUAGE_CODE_DO_TRANSLATION = "in-tl"
 
 # Supported languages
+# Note that the values of this list have also been manually copied into a
+# helper method in the main code.org repo; any changes made here should be
+# reflected in pegasus/helpers/page_helpers.rb:localized_lesson_plan_url_HACK
```

Comment:
nit: should `localized_lesson_plan_url_HACK` be `hacky_localized_lesson_plan_url` to match the method name?

Reply:
oh, yes! I originally named it with `_HACK`, but changed it after the linter complained